Python/thread_pthread.h

```diff
@@ -152,6 +152,24 @@ PyThread__init_thread(void)
  * Thread support.
  */
 
+typedef struct {
+    void (*func) (void *);
+    void *arg;
+} pthread_func;
```

Comment:
Please don't call this `pthread_something`. It makes it look like it is part of the POSIX threads API, while it isn't.